### DISCUSSION TITLE
Azure Enteprise App - Downstream SSO ?

### DISCUSSION THREAD

**Discussion by misiekmgo** [AzureCommunityPL/group#204] [Pomoc] [answered]
Cześć wszystkim w Nowym Roku :) 
Zagadka (przynajmniej dla mnie) - być może będziecie w stanie naprowadzić mnie na kierunek działań. 
Mamy zdefiniowaną w naszym Azure aplikację Enterprise (Ariba) i skonfigurowane SSO dla niej . Dostałem ostatnio zadanie aby i tu przytaczam fragment oryginalnej wypowiedzi z supportu Ariba: 
_"Please send us the XML’s for Your Company Downstream SSO configuration so that we can conclude our SSO set up"_ 
do czego jeszcze dołączone zostały dwa pliki XML. Niestety nic nie znalazłem na temat Downstream SSO w powiązaniu z Azure - może ktoś z Was ma wiedzę na ten temat ? 
Kiedy odpytałem się supportu o co dokładnie chodzi stwierdzili, że też nie wiedzą dokładnie ale:
_"I know that your team has made the set up of Ariba Sourcing SSO, maybe you can use this as a starting point to make the SSO set up for downstream?
The set up as to be done individually for each of the below realms
Parent URL:   http://yourcompany.procurement-eu.ariba.com
Child1 URL:   http://yourcompany-1.procurement-eu.ariba.com"_
Ale my mamy na razie tylko 1 app Ariba i jej URL (Sign-on URL) to: http://yourcompany.sourcing-eu.ariba.com - czy dobrze rozumiem, że należałoby zdefiniować w Azure kolejną aplikację z podanym wyżej ParentURLem ? Jak to spiąć z Child1URLem ?

**Comment by mgrabarz** [comment, marked as answer]
Jak dla mnie, to oni chcą od ciebie szczegółów dotyczących SAMLa, przy założeniu że go używasz np przez ADFS.

**Reply by misiekmgo** [reply]
Chyba właśnie przed chwilą się udało od nich więcej uzyskać info. Wygląda, że chcą abym po prostu stworzył dwie nowe aplikacje Ariba z nowymi Sign On URLami no i wtedy faktycznie będę im musiał zwrotnie podać informacji SAMLowe (używamy AD DS) + Login URL, Azure ID ident, Logotu URL - right ?

**Reply by pawelserwan** [reply]
wygląda na to że tak. Stwórz 2 nowe aplikacje w portalu Azure --> AAD--> Enterprise Apps. Po zdefiniowaniu apki przejdź do opcko Single Sign-On i tam mam możliwość wygenerowania XMLa z ustawieniami twojego IdP (AAD).

**Reply by misiekmgo** [reply]
Dokładnie tak zrobiłem - dzięki za wsparcie ! :)
